Shared/Scripts/bxl.ps1

```diff
@@ -449,7 +449,9 @@ Log -NoNewline "Building using the ";
 Log-Emphasis -NoNewline $($useDeployment.description)
 Log " version of BuildXL.";
 
-$AdditionalBuildXLArguments += "/environment:$($useDeployment.telemetryEnvironment)";
+$Nuget_CredentialProviders_Path = [Environment]::GetEnvironmentVariable("NUGET_CREDENTIALPROVIDERS_PATH");
+
+$AdditionalBuildXLArguments += "/environment:$($useDeployment.telemetryEnvironment) /unsafe_GlobalUntrackedScopes:$Nuget_CredentialProviders_Path";

```

Comment:
>$Nuget_CredentialProviders_Path [](start = 114, length = 31)

I thought this was supposed to be the environment variable's name, not its value